### ISSUE TITLE
Problems with @types/angularjs

### ISSUE THREAD

**Issue by dpogue** [Microsoft/TypeScript#9395]
**TypeScript Version:**  nightly (1.9.0-dev.20160627-1.0)

**Code**

```bash
npm install --save @types/angularjs
```

```ts
import * as angular from 'angular';
```

**Expected behavior:** No compiler errors.

**Actual behavior:** Could not find module 'angular'.

---

The reason this happens is because typings for several separate Angular npm packages are being published as a single types package with an incorrect name.
The name of the Angular npm module is `angular`, but TypeScript would need to look at `angularjs` to find the definition files.

There are also issues caused by referencing angular-mocks.d.ts (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angularjs/index.d.ts#L6) which tries to use types that haven't been defined by index.d.ts yet.

The correct solution is to split the single @types/angularjs package up into several @types/angular, @types/angular-mocks, @types/angular-animate, etc. packages.

/cc @RyanCavanaugh

**Comment by mhegazy** [comment]
just pushed a fix for this. republishing now. should be updated within the next few hours.
